tools/spell_checker/dictionary.txt

```diff
@@ -849,7 +849,6 @@ diverges
 divil
 dizzied
 dodecahedron
-doesnt
```

Comment:
I don't think you should be removing this as it didn't come from any of the changed lines in this PR.